validator/graffiti/parse_graffiti.go

```diff
@@ -27,6 +38,45 @@ func ParseGraffitiFile(f string) (*Graffiti, error) {
 	if err := yaml.Unmarshal(yamlFile, g); err != nil {
 		return nil, err
 	}
+
+	specific := make(map[types.ValidatorIndex]string, len(g.Specific))
+	for i, o := range g.Specific {
+		specific[types.ValidatorIndex(i)] = ParseHexGraffiti(o)
+	}
+
+	g.Specific = specific
+	g.Default = ParseHexGraffiti(g.Default)
+	g.Ordered = parseGraffitiStrings(g.Ordered)
+	g.Random = parseGraffitiStrings(g.Random)
 	g.Hash = hashutil.Hash(yamlFile)
 	return g, nil
 }
+
+// ParseHexGraffiti checks if a graffiti input is being represented in hex and converts it to ASCII if so
+func ParseHexGraffiti(rawGraffiti string) string {
+	splitGraffiti := strings.SplitN(rawGraffiti, ":", 2)
+	if strings.ToLower(splitGraffiti[0]) == "hex" {
```

Comment:
For maintenance purposes, it is easier to refactor this "magic" string to the top of the file as a variable, perhaps something called "hexGraffitiPrefixKey"